alacritty/src/window.rs

```diff
@@ -175,10 +202,22 @@ impl Window {
             } else {
                 let theme = AlacrittyWaylandTheme::new(&config.colors);
                 windowed_context.window().set_wayland_theme(theme);
+
+                let surface = windowed_context.window().wayland_surface().unwrap();
+                let proxy: Proxy<WlSurface> = unsafe { Proxy::from_c_ptr(surface as _) };
+                wayland_surface = Some(proxy.attach(wayland_event_queue.as_ref().unwrap().token()));
```

Comment:
Two short comments on each of these blocks might be useful? Since they do two completely different things unrelated to each other.